src/addAllAssetsToCompilation.js

```diff
@@ -31,7 +31,16 @@ function resolveOutput(compilation, addedFilename, outputPath) {
 async function addFileToAssets(
   compilation,
   htmlPluginData,
-  { filepath, typeOfAsset = 'js', includeSourcemap = true, hash = false, publicPath, outputPath, files = [] }
+  {
+    filepath,
+    typeOfAsset = 'js',
+    includeSourcemap = true,
+    includeGzipped = true,
```

Comment:
I'm not sure if I'd like an option for it. Might be surprising that it just silently doesn't add a file you've asked it to.

`skipMissing` or something?